README.md

```diff
@@ -1 +1,45 @@
-# contact-form
\ No newline at end of file
+# MyPortfolio Project
+This Project is created purposely to learn how to build forms with the use of html5 validations.
+
+
+## Built With
+* VsCode
+* HTML
+* CSS
+
+## Getting Started
+
+Prerequisites
+* Text editor
+* GitHub
+* Git
+
+Clone this repository on to your local PC
+
+* Click on the Code green button
+* By the right end of the read-only input containing the repository link click the clipboard icon to copy the link
+* In your local PC, open your terminal in the folder you would like to clone the repository into
+* Clone the repository with the command: git clone (copied link); like so: https://github.com/OyePriscilla/contact-form
+* After the clone, type in the command cd MyPortfolioSetup to access the directory on the terminal
+
+## Deployment
+Access the online version of this project through this link [Live Demo Link](https://oyepriscilla.github.io/contact-form)
+
+## Author
+👤 **Author1**
+👤 **Oyebade Prisicilla**
+
+- GitHub: [@OyePriscilla](https://github.com/OyePriscilla)
+- Twitter: [@Prisicilla](https://twitter.com/Prisicilla)
+- LinkedIn: [Priscilla](https://linkedin.com/in/Priscilla)
+
+## Contibution
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Acknowledgments
+Hat tip to the person that reviewed my work! Thanks alot.
+
+## License
+This project is MIT licensed.
```

Comment:
- [OPTIONAL] Kindly link the license file so it refers to the corresponding file in your repo. 
**Ex:** `This project is [MIT](./MIT.md) licensed.`